src/wallet/wallet2.cpp

```diff
@@ -7881,6 +7876,18 @@ void wallet2::get_outs(std::vector<std::vector<tools::wallet2::get_outs_entry>>
           error::get_output_distribution, "Daemon reports suspicious number of rct outputs");
     }
 
+    std::vector<uint64_t> output_blacklist;
+    if (bool get_output_blacklist_failed = !get_output_blacklist(output_blacklist))
```

Comment:
Do we need a variable here (its value is always false)?

Reply:
I put it into a variable so that you get the variable name stringified in the error, although you could argue that the throw exception's description is descriptive enough to be able to tell what happened.

Or more so you could put the function in the throw macro, but I shy away from putting functions in assert like macros that look like they can be compiled out.